azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -462,6 +464,9 @@ private void loadQueuedFlows() throws ExecutorManagerException {
     List<Integer> executionIds = new ArrayList<Integer>();
     executionIds.addAll(getRunningFlowsHelper(projectId, flowId,
       queuedFlows.getAllEntries()));
+    if(runningCandidate != null) {
```

Comment:
Add space between if and (.

You may want to consider letting IDE format the modified methods.